### PR TITLE
Wrap `sqlite3_exec` for libsqlite3

### PR DESCRIPTION
**Pull request by Coekjan**

### CODE REVIEW

**Comment by Coekjan** [comment]
I imitated the callback function processing methods in other wrapped libraries.

Please review my changes. If I am wrong, kindly let me know.

**Comment by ksco** [comment]
IIRC wpp uses this function, can you verify if that application works with wrapped libsqlite3 now?

**Comment by Coekjan** [comment]
> IIRC wpp (wps presentation) uses this function, can you verify if that application works with wrapped libsqlite3 now? btw PR itself looks good to me.

Seems that WPP works good.

**Review by ksco — APPROVED** [review]

File: src/wrapped/wrappedlibsqlite3.c [hunk 1]
```diff
@@ -25,8 +25,40 @@ const char* libsqlite3Name = "libsqlite3.so.0";
...
+GO(0)   \
+GO(1)   \
+GO(2)   \
+GO(3)   \
+GO(4)
+
+// sqlite3_exec ...
+#define GO(A) \
+static uintptr_t my_sqlite3_exec_fct_##A = 0; \
+static int* my_sqlite3_exec_##A(void* a, int b, char** c, char** d) { \
```
Comment (ptitSeb) [inline src/wrapped/wrappedlibsqlite3.c:40]: Type of the function is
```C
int sqlite3_exec(
  sqlite3*,                                  /* An open database */
  const char *sql,                           /* SQL to be evaluated */
  int (*callback)(void*,int,char**,char**),  /* Callback function */
  void *,                                    /* 1st argument to callback */
  char **errmsg                              /* Error msg written here */
);
```
return type if the callback is `int` not `int*`

**Review by ptitSeb — CHANGES_REQUESTED** [review]

File: src/wrapped/wrappedlibsqlite3.c (hunk 1 above)
Comment (Coekjan) [reply src/wrapped/wrappedlibsqlite3.c]: Sorry for my mistake. I have fixed it in my newest commit.

**Review by ptitSeb — APPROVED** [review]

**Pull request merged**